docs/sources/installation/debian.md

```diff
@@ -64,6 +64,9 @@ which is officially supported by Docker.
 3. Install Docker using the get.docker.com script:
  
     `curl -sSL https://get.docker.com/ | sh`
+    > **tips** If your company is behind a filtering proxy, you may find the apt-key command fails for the Docker repo during isntalllation. To work around this, add the key directly using the following:
```

Comment:
Hi thanks for the PR.  Couple of things:
- wrap lines at 80 characters please
- we prefer notes over tips
- there was a spelling error "installation"

https://gist.github.com/moxiegirl/6b71ed4941952d689383

-->Fix:

>**Note**: If your company is behind a filtering proxy, you may find the `apt-key`
>command fails for the Docker repo during installation. To work around this,
>add the key directly using the following:
>
> 	    $ wget -qO- https://get.docker.com/gpg | sudo apt-key add -